conf/node-default.properties

```diff
@@ -89,10 +89,10 @@
 # DB.LockTimeout = 60
 
 ## Maximum number of rows to be inserted in a single SQL statement.
-## Defaults to 10000, should be ok in most situations.
+## Defaults to 1000, should be ok in most situations.
 ## May be fine-tuned according to your DBMS or your machine's performance.
 ## Warning: a high value (> 15000 rows) is known to generate queries too big for an SQLite backend
-# DB.InsertBatchMaxSize = 10000
+# DB.InsertBatchMaxSize = 1000
```

Comment:
this parameter is sketchy... maybe we should hard code and not make it configurable as it can make the node to fail...